### DISCUSSION TITLE
PLANS

### DISCUSSION THREAD

**Discussion by Muhammad-Sarem-Tahir** [Coder-Boiiiiiii/Project-Mock#2] [Announcements]
# 👋Introduction👋
Hey guys!
Hope you've all been well, and have been studying up for the exams!
Anyways, this post is all about my plans for this project (for now).

# Planned Updates (v1.2 & so on...)

Lets start with the planned updates which are most likely to be implemented!
We'll go from highly likely to maybe, let's begin!

## Highly Likely 👍👍

1. More years of papers to increase variety of papers.
2. Paper 2 questions.
3. Mixed questions from both; Paper 2 & 4.

## Likely 👍

1. Error & message outputs when wrong input was used.
2. Slide/Step counter.
3. Mock paper properties displayed at end screen.

## Somewhat Likely 😐

1. Different board papers (like AQA & Edexcel) which can be mixed with each other or have separately.
2. Difficulty Levels

## Maybe 😑

1. Customizations for UI.
2. Using sliders and other more user friendly UI components rather than input boxes.
3. Addition of a new option which has EXTREMELY old questions (we talking the 90's)

# The Real Question 🤔

The real question however is... WHEN WILL I ACTUALLY IMPLEMENT ALL OF THIS?!
Well, simple answer is that it'll take time and every update will be a bit smaller because of the limited amount of time I will be able to put into the project.
Why you might ask, well its because I need to study for my own exams which is why I will not be able to invest lots of time (like how I did before).

But don't worry!
I will try my best to update this and also try not to forget about this!

# The TL;DR (for the lazy bum bums) 📝

To conclude, I will not be able to update this whole thing as often as before because of many reasons, the biggest one being the IGCSEs.
I also gave a list of the updates which might get added, from the addition of paper 2 to the addition of different boards!

Anyways, 
That's all from me,
Bye!
